tests/data/dataset/test_composite.py

```diff
@@ -0,0 +1,81 @@
+from typing import Sequence
+from typing import Tuple
+
+import hypothesis.strategies as st
+import numpy as np
+import torch
+from hypothesis import given
+from myrtlespeech.data.dataset.composite import Composite
+from torch.utils.data import Dataset
+
+
```

Comment:
The other `test_*.py` files have comments in to separate out the mocks/fixtures/strategies, may be useful here also:

```python3

# Fixtures and Strategies -----------------------------------------------------


# Tests -----------------------------------------------------------------------


```